docs/index.md

```diff
@@ -1,20 +1,20 @@
 ---
-organization: wedtm
-category: ["internet"]
-icon_url: "/images/plugins/wedtm/hibp.svg"
+organization: turbot
+category: ["security"]
+icon_url: "/images/plugins/turbot/hibp.svg"
 brand_color: "#3A9AC4"
-display_name: Security
+display_name: Have I Been Pwned
 name: hibp
-description: Steampipe plugin to query Have I Been Pwned breaches, pastes, and passwords
-og_description: Query compromised data with SQL! Open source CLI. No DB required.
-og_image: "TBD"
+description: Steampipe plugin to query breaches, account breaches, pastes and passwords from Have I Been Pwned.
+og_description: Query HIBP data with SQL! Open source CLI. No DB required.
+og_image: "/images/plugins/turbot/hibp-social-graphic.png"
 ---
 
-# HaveIBeenPwned? + Steampipe
+# Have I Been Pwned? + Steampipe
```

Comment:
```suggestion
# Have I Been Pwned + Steampipe
```

Not sure if we should include the `?` or not in titles here, README, etc., but we primarily need to be consistent, so if other places don't include `?`, we can leave it out here too